themes/question/image_select/survey/questions/answer/listradio/rows/answer_row.twig

```diff
@@ -43,13 +42,12 @@ question_template_attribute.horizontal_scroll   : {{question_template_attribute.
             id="answer{{ name}}{{ code }}"
             value="{{code}}"
             {{ checkedState }}
-            onclick='cancelBubbleThis(event); {{ sCheckconditionFunction }}'
         />
-        <label for="answer{{ name }}{{ code }}" class="imageselect-label" onclick='cancelBubbleThis(event); {{ sCheckconditionFunction }}'>
+        <label for="answer{{ name }}{{ code }}" class="imageselect-label">
 
-         <img class="unforce-height" src="{{ answer }}" 
-         style="{%if question_template_attribute.fix_width > 1%} 
-            width: {{question_template_attribute.fix_width}}px 
+         <img class="unforce-height" src="{{ answer }}"
```

Comment:
Just a note, if possible, avoid white-space changes, or do white-space changes in a separate PR. It will make reviewing the code easier. Thank you. :)